### DISCUSSION TITLE
🚀 Feature request: Field collapse caret icon should be more visible

### DISCUSSION THREAD

**Discussion by jaredchesebro** [ExpressionEngine/ExpressionEngine#4136] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
I find that clients often have trouble with how small the collapse icons are for fields.

![field-collapse](https://github.com/ExpressionEngine/ExpressionEngine/assets/21958491/41ee1cfe-be69-4648-b648-51721e66fdb7)

**Describe the solution you'd like**
Make it larger or redesign the collapse UI/UX would improve this feature a lot. Any redesign should try to meet WCAG 2.1 standards.

**Comment by blue-dreamer** [comment]
Agreed, the icon is far too small and the colour is virtually invisible, for me at least. Going through the CP and making the UI fully WCAG accessible would be plus for for EE.
